src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -111,7 +111,7 @@ internal void GetDeclarationDiagnostics(DiagnosticBag addTo)
 
                 if (typeParam.HasUnmanagedTypeConstraint)
```

Comment:
>if (typeParam.HasUnmanagedTypeConstraint) [](start = 16, length = 41)

It feels like typeParam.ForceComplete is the right place to report this error.

Reply:
Consolidated into the other comment.

---
In reply to: [177237646](https://github.com/dotnet/roslyn/pull/25680#discussion_r177237646) [](ancestors = 177237646)